pyscriptjs/src/main.ts

```diff
@@ -242,7 +254,28 @@ from pyscript import micropip, Element, console, document`);
         for (let i = 0; i < paths.length; i++) {
             logger.info(`  fetching path: ${fetchPaths[i]}`);
             try {
-                await runtime.loadFromFile(paths[i], fetchPaths[i]);
+                const filePromise = fetch(fetchPaths[i]);
```

Comment:
Just tested both and seems like we can use `robustFetch` here as well, might be a good idea to use it since we can handle non 200 errors.

Would it be possible to add a test for this just to make sure it doesn't blow up some other way?


```suggestion
                const filePromise = robustFetch(fetchPaths[i]);
```